packages/foam-vscode/src/features/dataviz.ts

```diff
@@ -20,33 +20,40 @@ const feature: FoamFeature = {
     });
 
     vscode.commands.registerCommand("foam-vscode.show-graph", async () => {
-      const foam = await foamPromise;
-      panel = await createGraphPanel(foam, context);
-      const onFoamChanged = _ => {
-        updateGraph(panel, foam);
-      };
-
-      const noteAddedListener = foam.notes.onDidAddNote(onFoamChanged);
-      const noteUpdatedListener = foam.notes.onDidUpdateNote(onFoamChanged);
-      const noteDeletedListener = foam.notes.onDidDeleteNote(onFoamChanged);
-      panel.onDidDispose(() => {
-        noteAddedListener.dispose();
-        noteUpdatedListener.dispose();
-        noteDeletedListener.dispose();
-        panel = undefined;
-      });
+      if (panel) {
+        const columnToShowIn = vscode.window.activeTextEditor
+        ? vscode.window.activeTextEditor.viewColumn
+        : undefined;
```

Comment:
what does `undefined` do here? Could it make sense to use the `vscode.ViewColumn.Two` value, which is what we use when we create the panel the first time?

Reply:
I'll be honest, I took it from here https://code.visualstudio.com/api/extension-guides/webview#webviews-api-basics
However, I don't think we should default to a value since the column might have changed, as I do in the example.

Reply:
sounds good